### PR TITLE
Add scirnaSeq to Assays

### PR DESCRIPTION
**Pull request by pitviper6**

### CODE REVIEW

File: terms/experimentalData/assay.json [hunk 1]
```diff
@@ -413,6 +413,11 @@
             "description": "A method for genome-wide CpG island (CGI) methylation sequencing for single cells (scCGI-seq), combining methylation-sensitive restriction enzyme digestion and multiple displacement amplification for selective detection of methylated CGIs",
             "source": "https://www.ncbi.nlm.nih.gov/pubmed/28126923"
         },
+        {
+            "const": "scirnaSeq",
+            "description": "The sci-RNA-seq uses the combinatorial indexing to identify single cells without single cell isolation. Two-level indexing (RT barcode + PCR barcodes (i5 + i7)) or three-level indexing (RT barcode + PCR barcodes (i5 + i7) + Tn5 barcodes) can be used. Three-level indexing is a bit more difficult since you need to assemble many indexed Tn5 transposomes.",
+            "source": "http://www.ebi.ac.uk/efo/EFO_0010550"
+        }.
```
Comment (kelshmo) [inline terms/experimentalData/assay.json:420]:
```suggestion
        },
```

**Review by kelshmo — CHANGES_REQUESTED** [review]
The term and description looks good. There is a typo in the format, I have proposed the change from . to ,. I also believe the id # needs to be [incremented here](https://github.com/Sage-Bionetworks/synapseAnnotations/blob/b9ea3238206da5e677994686e0396605966761df/terms/experimentalData/assay.json#L3). Can you confirm this @avanlinden ?

**Comment by avanlinden** [comment]
@kelshmo @pitviper6 Correct, the version id should be incremented as well!

**Comment by pitviper6** [comment]
Thanks! By incremented, do I add an increment of one to the number (aka,
0.0.12 to 0.0.13?

**Comment by avanlinden** [comment]
> Thanks! By incremented, do I add an increment of one to the number (aka, 0.0.12 to 0.0.13?

Yep!

**Review by Aryllen — APPROVED** [review]
I incremented the assay version.

**Comment by Aryllen** [comment]
@kelshmo, will you please check over and approve?

**Review by kelshmo — APPROVED** [review]

**Pull request merged**